scripts/pre_commit_linter.py

```diff
@@ -1321,12 +1335,12 @@ def _check_js_component_name_and_count(self):
             print ''
             return summary_messages
 
-    def _check_js_file_overview(self):
-        """This function checks that all JS files contain their respective
-        file overview.
+    def _check_mandatory_js_patterns(self):
```

Comment:
No, please go along this comment:

> This file overview check can be converted to a simple regex check. We could then create a list of MANDATORY_PATTERNS_JS_REGEXP, MANDATORY_PATTERNS_PYTHON_REGEXP etc. The copyright notice check here uses a regex at its core and can thus be added to the python regexp list above. In this way, we will need only a single function for all such mandatory regex checks. This will make the check efficient.

That is, create a single function for checking mandatory regexp and merge both the copyright notice check and this check.

Reply:
Added the change.
PTAL!
Thanks.